modules/features/search/src/main/java/au/com/shiftyjelly/pocketcasts/search/component/ImprovedSearchEpisodeResultRow.kt

```diff
@@ -0,0 +1,215 @@
+                remember(duration, context) { TimeHelper.getTimeDurationMediumString((duration * 1000).toInt(), context) }
+            val dateFormatter = RelativeDateFormatter(context)
+            val formattedPublishDate = remember(publishedAt, dateFormatter) { dateFormatter.format(publishedAt) }
+
+            TextC70(
+                fontSize = 11.sp,
+                text = formattedPublishDate,
+                maxLines = 1,
+            )
+            TextH40(
+                text = title,
+                color = MaterialTheme.theme.colors.primaryText01,
+                maxLines = 1,
+            )
+            TextH60(
+                fontSize = 12.sp,
+                text = formattedDuration,
+                color = MaterialTheme.theme.colors.secondaryText02,
+                fontWeight = FontWeight.W600,
+                maxLines = 1,
+            )
+        }
+        episode?.let {
+            val buttonColor = MaterialTheme.theme.colors.primaryInteractive01.toArgb()
+            AndroidView(
+                modifier = Modifier.size(48.dp),
+                factory = {
+                    PlayButton(it).apply {
+                        listener = playButtonListener
+                    }
+                },
+                update = { playButton ->
+                    playButton.setButtonType(episode, buttonType = PlayButtonType.PLAY, color = buttonColor, null)
+                },
+            )
+        }
+    }
+}
+
+@Preview
+@Composable
+private fun PreviewEpisodeResultRow(
+    @PreviewParameter(ThemePreviewParameterProvider::class) themeType: Theme.ThemeType,
+) {
+    AppThemeWithBackground(themeType) {
+        ImprovedSearchEpisodeResultRow(
+            episodeUuid = "",
+            podcastUuid = "",
+            title = "Episode title",
+            duration = 320.0,
+            publishedAt = Date(),
+            playButtonListener = object : PlayButton.OnClickListener {
+                override var source: SourceView = SourceView.SEARCH_RESULTS
+
+                override fun onPlayClicked(episodeUuid: String) {
+                    TODO("Not yet implemented")
+                }
+
+                override fun onPauseClicked() {
+                    TODO("Not yet implemented")
+                }
+
+                override fun onPlayNext(episodeUuid: String) {
+                    TODO("Not yet implemented")
+                }
+
+                override fun onPlayLast(episodeUuid: String) {
+                    TODO("Not yet implemented")
+                }
+
+                override fun onDownload(episodeUuid: String) {
+                    TODO("Not yet implemented")
+                }
+
+                override fun onStopDownloading(episodeUuid: String) {
+                    TODO("Not yet implemented")
+                }
+
+                override fun onPlayedClicked(episodeUuid: String) {
+                    TODO("Not yet implemented")
+                }
+            },
```

Comment:
```suggestion
            playButtonListener = object : PlayButton.OnClickListener {
                override var source: SourceView = SourceView.SEARCH_RESULTS
                override fun onPlayClicked(episodeUuid: String) {}
                override fun onPauseClicked() {}
                override fun onPlayNext(episodeUuid: String) {}
                override fun onPlayLast(episodeUuid: String) {}
                override fun onDownload(episodeUuid: String) {}
                override fun onStopDownloading(episodeUuid: String) {}
                override fun onPlayedClicked(episodeUuid: String) {}
            },
```
We probably don't need to add the TODO for the preview as we aren't going to add the implementation.